assemble/bin/accumulo-cluster

```diff
@@ -471,6 +471,12 @@ function control_services() {
           fi
         fi
       done
+      if [[ $operation == "stop" || $operation == "kill" ]]; then
+        echo "Cleaning tablet server entries from zookeeper for resource group $group"
+        if ! isDebug; then
+          "$accumulo_cmd" org.apache.accumulo.server.util.ZooZap -tservers -group "$group"
+        fi
```

Comment:
could `debugOrRun` be used here? Thinking it could print the command that would be run in debug mode.

Reply:
We could do that. I surveyed the other places in `accumulo-cluster` and we are not using `debugOrRun` in all cases where we are executing a command.

Reply:
I applied this change in 18844f8.

Reply:
`./accumulo-cluster stop --tservers=extra --local --dry-run`
returned: 
```
bin/accumulo' 'org.apache.accumulo.server.util.ZooZap' '-verbose' '-tservers' '-group' 'extra'
```

Reply:
@ddanielr - are you suggesting there is an issue? or were you just answering @keith-turner 's question about what it would look like?

Reply:
I'm just answering @keith-turner 's question. The debug option seems to work well!